liquidcore/config/views.py

```diff
@@ -0,0 +1,167 @@
+        permission_classes=[IsAdminOrSelf],
+        url_name='password'
+    )
+    def change_password(self, request, pk=None):
+        user = self.get_object()
+        serializer = PasswordChangeSerializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        data = serializer.validated_data
+
+        if not request.user.is_staff:
+            # admin users don't need to specify the old password
+            if not user.check_password(data['old_password']):
+                return Response({"old_password": "Invalid Password"},
+                                status=status.HTTP_400_BAD_REQUEST)
+        user.set_password(data['new_password'])
+        user.save()
+        return Response(status=status.HTTP_200_OK)
+
+
+class ServiceViewSet(viewsets.ReadOnlyModelViewSet):
+    queryset = Service.objects.all()
+    serializer_class = ServiceSerializer
+
+    @detail_route(
+        methods=['put'],
+        permission_classes=[IsAdminUser],
+        url_name='enabled'
+    )
+    def set_enabled(self, request, pk=None):
+        """Sets the service as enabled or disabled."""
+        serializer = ServiceEnabledSerializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        data = serializer.validated_data
+
+        service = self.get_object()
+        service.is_enabled = data['is_enabled']
+        service.save()
+        # TODO send start/stop command to system
+        return Response(status=status.HTTP_200_OK)
+
+class NodeViewSet(viewsets.ReadOnlyModelViewSet):
+    queryset = Node.objects.all()
+    serializer_class = NodeSerializer
+
+    @detail_route(
+        methods=['put'],
+        permission_classes=[IsAdminUser],
+        url_name='trusted'
+    )
+    def set_trusted(self, request, pk=None):
+        """Sets the service as enabled or disabled."""
+        serializer = NodeTrustedSerializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        data = serializer.validated_data
+
+        node = self.get_object()
+        node.is_trusted = data['is_trusted']
+        node.save()
+        # TODO send updated node list to the discovery service
+        return Response(status=status.HTTP_200_OK)
+
+
+@api_view()
+def network_status(request):
+    return Response({})
+
+class NetworkDomain(APIView):
+    def get(self, request, format=None):
+        return Response({"domain": settings.LIQUID_DOMAIN})
+
+    def put(self, request, format=None):
+        serializer = NetworkDomainSerializer(data=request.data)
+        if serializer.is_valid():
+            # TODO send change domain command to system
+            return Response({"detail": "not implemented"},
+                            status=status.HTTP_501_NOT_IMPLEMENTED)
+        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+class NetworkSettingAPIView(APIView):
+    def get(self, request, format=None):
+        setting = get_object_or_404(Setting, name=self.setting_name)
```

Comment:
Can we return a blank value instead of 404? Maybe use `get_or_create` to create a default value.